src/components/Footer.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const Footer = () => <footer>Footer text</footer>;
```

Comment:
Whatever message/copyright you would like to display in the footer, feel free to let me know anytime you want. Footer can vanish if you feel there is no need for it.

Reply:
Feels like a good place for attribution to the front-end designer.